ckanext/restricted/action.py

```diff
@@ -75,7 +75,8 @@ def body_from_user_dict(user_dict):
         mail_recipient(name, email, subject, body)
 
     except MailerException as mailer_exception:
-        log.error('Cannot send mail after registration')
+        log.error('Cannot send mail after registration for user {0} with email {1}'
+                  .format(user_dict.get('name'), user_dict.get('email')))
```

Comment:
You also removed the exc_info. With it the the `mailer_exception` can be removed all together.

Reply:
@jonathansberry, I used .format() because the rest of this code uses it, I don't want to switch styles from line to line in the same file, especially that we still run CKAN tests on Python 2 for this extension.
@tomeksabala I don't understand. Do we need exc_info here? I just wanted to log the information we need, which is username and email address that caused an SMTP error.

Reply:
Re 1st. Good call Michał.
Re 2nd: exc_info will add exception details to the message. I was missing this part while reviewing the error in sentry last time. It is logged in a separate error message with bad formatting but this other entry was hard to find for me.
Logging all of the information in the error message (user friendly message + exception details) is the way to go IMO.
exc_info is automatically added in `log.exception()` which even better could be used here.

Reply:
got it, that makes sense